git-sync/ssh.md

```diff
@@ -0,0 +1,65 @@
+# Using SSH with git-sync
+
+Git-sync supports using the SSH protocol for pulling Git content.
+
+For now, only keys without password protection are supported.
+
+## Setup
+
+Write a config file for a Secret with name "git-creds" that holds your SSH private key, with the key mapped to "id-rsa".
+```
+{
+  "kind": "Secret",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "git-creds",
+    "namespace": "kube-system"
```

Comment:
Do the creds always need to go into kube-system for this to work? If not, I'd leave out the namespace.